/test pull-kueue-verify-main

---

Sorry that it's caused by my fault and I overlook it at the first glance, wasting CI resources
```go
level=warning msg="[runner/exclusion_rules] Skipped 0 issues by rules: [Path: \"zz_generated\\\\..+\\\\.go$\", Linters: \"gci, golines\"]"
pkg/dra/counters_test.go:506:1: File is not properly formatted (golines)
				_, _, errs := prepareCounterCharge(ctx, cl, "valid-device-class", []resourcev1.DeviceSelector{{CEL: &resourcev1.CELDeviceSelector{Expression: "device.driver =="}}}, make(map[string]*resourcev1.DeviceClass), claimPath, 1)
^
1 issues:
* golines: 1
make-timing: started=2026-08-05T17:07:40Z finished=2026-08-05T17:11:42Z status=123 duration=242s command=find . \( -path ./site -o -path ./bin -o -path ./vendor \) -prune -false -o -name go.mod -exec dirname {} \; | xargs -P 0 -n 1 sh -c ' cd "$1" || exit 1; dirs=$(go list -f "{{.Dir}}" ./... | grep -vE "/(client-go|internal/mocks)(/|$)" | ...
make[1]: *** [Makefile-verify.mk:196: verify-ci-lint] Error 123
make[1]: Leaving directory '/home/prow/go/src/sigs.k8s.io/kueue'
make-timing: started=2026-08-05T17:03:48Z finished=2026-08-05T17:11:42Z status=2 duration=474s command=make -j 8 --output-sync=target verify-checks
make: *** [Makefile-verify.mk:74: verify] Error 2
```

--- a/pkg/dra/counters_test.go
+++ b/pkg/dra/counters_test.go
@@ -503,7 +503,15 @@ func TestSelectorErrorPaths(t *testing.T) {
 		{
 			name: "request selector compilation error",
 			run: func() field.ErrorList {
-				_, _, errs := prepareCounterCharge(ctx, cl, "valid-device-class", []resourcev1.DeviceSelector{{CEL: &resourcev1.CELDeviceSelector{Expression: "device.driver =="}}}, make(map[string]*resourcev1.DeviceClass), claimPath, 1)
+				_, _, errs := prepareCounterCharge(
+					ctx,
+					cl,
+					"valid-device-class",
+					[]resourcev1.DeviceSelector{{CEL: &resourcev1.CELDeviceSelector{Expression: "device.driver =="}}},
+					make(map[string]*resourcev1.DeviceClass),
+					claimPath,
+					1,
+				)
 				return errs
 			},
 			wantField: reqPath.Child("exactly", "selectors").Index(0).Child("cel", "expression").String(),

